src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -112,6 +113,10 @@
     @Column(name = "example_result")
     private Boolean exampleResult;
 
+    // For Exam exercises, we need to track whether it was corrected for a second time
+    @Column(name = "isAssessedTwice")
```

Comment:
I don't like this database extension! It's not extendable for an actual implementation of a second correction (an actual second correction would need a second result). 

It also does not make sense for most results in the Artemis database and we would need to remove it again in the future.

Reply:
I removed the `isAssessedTwice` flag and all its corresponding functionality from the client and server in 41bb190